game/templates/game/landing.html

```
@@ -444,116 +510,46 @@ <h4>Legal</h4>
-                      <div>
-                          <h2 style="color:#fff; font-size:1.05rem; font-weight:600; margin-bottom:8px;">Disclaimer of Liability</h2>
-                          <p style="color:rgba(255,255,255,0.7); font-size:0.92rem; line-height:1.7; margin:0;">The platform services and integrated AI engines are offered on an "as-is" operational basis. The maintenance team assumes no liability for match state disruptions, profile cache rollbacks during server updates, or local dependency failures in client-side setups.</p>
-                      </div>
-                  </div>
-  
-                  <!-- Last Updated -->
-                  <p style="color:#64748b; font-size:0.82rem; margin-top:6px;">Last updated: May 2026</p>
-  
-              </div>
-          </div>
+    </section>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </section> ]

(tag-pair)